imports/plugins/included/product-detail-simple/client/containers/productDetailContainer.js

```diff
@@ -68,9 +77,9 @@ class ProductDetailContainer extends Component {
         });
       } else {
         productId = currentProduct._id;
```

Comment:
where is `productId` initialized? If none, then you've made it a global variable which can pollute your global namespace